src/main/scala/ba/sake/hepek/bulma/component/PaginationComponents.scala

```diff
@@ -0,0 +1,39 @@
+package ba.sake.hepek.bulma.component
+
+import ba.sake.hepek.bulma._
+import scalatags.Text.all._
+
+object PaginationComponents extends PaginationComponents
+
+sealed trait PaginationElement extends BulmaElement
+
+case class Pagination(previous: Option[String], next: Option[String]) extends BulmaElement {
+  override def content = frag(
+    previous.fold[Frag](SeqFrag[String](List()))(a(cls := "pagination-previous")(_)),
```

Comment:
You don't have to `fold` all these frags. Scalatags can render an `Option[Frag]` with no problem.  
Just write `previous.map(a(cls := "pagination-previous")(_))`

Reply:
thanks for mentioning that! Fixed.